src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -34,11 +34,15 @@ import {
 import { GET_TASK_TESTS } from "gql/queries";
 import { useUpdateURLQueryParams, useNetworkStatus } from "hooks";
 import { TestStatus, RequiredQueryParams, TableOnChange } from "types/task";
-import { queryString, url, string } from "utils";
+import { queryString, url, string, environmentalVariables } from "utils";
 
+const { getLobsterURL } = environmentalVariables;
 const { msToDuration } = string;
 const { getPageFromSearch, getLimitFromSearch } = url;
 const { parseQueryString, queryParamAsNumber } = queryString;
+
+const DEPRECATED_LOGKEEPER_LOBSTER_URL = "https://logkeeper.mongodb.org";
```

Comment:
Sorry i meant it should live here https://github.com/evergreen-ci/spruce/blob/11ed5b90d559a99e3d910d6f76c129e6d5c220cb/src/constants/externalResources.ts